src/SWP/UpdaterBundle/README.md

```diff
@@ -0,0 +1,2 @@
+# SWPUpdaterBundle
+Provides automatic Symfony2 apps upgrades
```

Comment:
integrates symfony2 with ahilles107/updater library wutch provide easy way for php applications updates.